pymc3/step_methods/hmc/nuts.py

```diff
@@ -110,7 +130,17 @@ def astep(self, q0):
 
         self.m += 1
 
-        return q
```

Comment:
Do you mean the differences of the hamiltonians? I've got a half finished patch that does that and also adds the information if the transition was diverging and the maximal divergence along the trajectory. But I'd rather make that a separate pull request. It makes changes in the nuts code itself.